metametamodel/metametamodel.adoc

```diff
@@ -139,11 +139,12 @@ A Concept `implements` zero or more <<ConceptInterface, ConceptInterfaces>>.
 A Concept can have any number of <<Classifier.features, `features`>>, given it is a <<Classifier>>.
 
 .Constraints
-TBD
+A Concept MUST NOT extend itself, or form circles via `extends`.
 
 [[Annotation]]
 ==== Annotation
-An Annotation is a limited[*] additional, orthogonal information attached to potentially any node, sharing the node’s lifecycle.
+An Annotation is a limitedfootnote:["limited" because the more complex the annotation is, the more likely it should not be part of the annotated node's lifecycle -- We might want to reuse the complex annotation somewhere else.
```

Comment:
Proposal: "[...]is an additional piece of information[...]", and then explain limited + orthogonal in a separate sentence. That way, you probably don't even need the footnote.

Reply:
Good idea, what do you think of this attempt?